py/torch_migraphx/fx/tracer/acc_tracer/acc_ops.py

```diff
@@ -373,6 +374,27 @@ def clamp(*, input, min=None, max=None):
     return torch.clamp(input=input, min=min, max=max)
 
 
+@register_acc_op_mapping(
```

Comment:
```suggestion
if 'torchvision' in sys.modules:
    @register_acc_op_mapping(
```
wrap the roi_align function definition in this if so that it doesnt blow up if torchvision isnt installed